Source/WebKit/UIProcess/Cocoa/GroupActivities/GroupActivitiesCoordinator.mm

```diff
@@ -55,48 +55,54 @@ - (id)initWithParent:(WebKit::GroupActivitiesCoordinator&)parent
 
 -(void)playbackCoordinator:(AVDelegatingPlaybackCoordinator *)coordinator didIssuePlayCommand:(AVDelegatingPlaybackCoordinatorPlayCommand *)playCommand completionHandler:(void (^)(void))completionHandler {
     dispatch_async(dispatch_get_main_queue(), ^{
-        if (_parent) {
-            _parent->issuePlayCommand(playCommand, [completionHandler = makeBlockPtr(completionHandler)] {
-                completionHandler();
-            });
-        }
+        RefPtr parent = _parent.get();
+        if (!parent)
+            return;
```

Comment:
Not new to your PR but I'd be surprised if it was OK to NOT call the completionHandler here..